x/evm/keeper/msg_server.go

```diff
@@ -445,7 +445,7 @@ func (s msgServer) CreateDeployToken(c context.Context, req *types.CreateDeployT
 		return nil, err
 	}
 
-	if err = s.nexus.RegisterAsset(ctx, chain, nexus.NewAsset(req.Asset.Name, false), sdk.ZeroUint()); err != nil {
+	if err = s.nexus.RegisterAsset(ctx, chain, nexus.NewAsset(req.Asset.Name, false), dailyMintLimit); err != nil {
```

Comment:
Using `dailyMintLimit` here would mean we need to make sure the window hardcoded in the contract and default in core always stay the same?